artifacthub/library/general/disallowanonymous/1.0.1/artifacthub-pkg.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.0.1
```

Comment:
FYI: I'm guessing this `1.0.1` folder is stale from earlier, and probably should be removed.

Reply:
I see you just did!